web/styles/compose.css

```diff
@@ -1209,8 +1211,10 @@ textarea.new_message_textarea {
         & .enter_sends_choice_radio {
             width: auto;
             cursor: pointer;
-            margin: 4px 0 0;
+            height: 1em; /* 14px at 14px em */
```

Comment:
In lieu of an em conversion (1em doesn't say much), it might be useful to document that we set the height this way to allow the radio button to scale.

Reply:
Added this comment, is this what you're thinking?

```css
            /* The height is 14px at 14px em, but we measure using em to allow
               the radio button size to scale. */
            height: 1em; 
```

Reply:
Ah I see, sure. Thanks for clarifying what you meant